GoogleChromeLabs/wp-sitemaps
# #67 Add an XML Stylesheet

### Issue Number
https://github.com/GoogleChromeLabs/wp-sitemaps/issues/67 - Add an XML Stylesheet (XSL file)
This also addresses https://github.com/GoogleChromeLabs/wp-sitemaps/issues/77  - Make CSS for XSL file filterable

### Description
Adds an .xsl stylesheets file to style all of the sitemaps 

### Type of change
Please select the relevant options:


### Steps to test
View any of the stylesheets listed below to see them laid out in a table with some basic styles applied
- http://your-local/sitemap.xml
- http://your-local/sitemap-posts-post.xml
- http://your-local/sitemap-taxonomies-category.xml
- http://your-local/sitemap-users.xml
(You may need to flush your redirects locally)

### Screenshots

**Sitemap Index**

<img width="1132" alt="Screenshot 2019-11-21 at 17 35 14" src="https://user-images.githubusercontent.com/182015/69362242-0fe46a00-0c86-11ea-907d-1219bb4fbdba.png">

**Pages Sitemap**

<img width="1132" alt="Screenshot 2019-11-21 at 17 35 00" src="https://user-images.githubusercontent.com/182015/69362243-107d0080-0c86-11ea-9c99-718fd0b93604.png">

### Acceptance criteria
